motioneye/utils.py

```diff
@@ -67,6 +67,15 @@
     (1920, 1080)
 ]
 
+RPI_MMAL_RESOLUTIONS = [
+    (640, 480),
+    (1280, 720),
+    (1280, 960),
+    (1920, 1080),
+    (1920, 1088),
+    (2592, 1944)
+]
+
```

Comment:
Are we limited to these precise resolutions when using MMAL? Why can't we use any of the other common resolutions?